src/Host/Client/Impl/Session/RSession.cs

```diff
@@ -488,6 +496,19 @@ public Task StartHostAsync(RHostStartupInfo startupInfo, IRSessionCallback callb
 "));
         }
 
+        private static Task SuppressUI(IRExpressionEvaluator eval) {
+            // # Suppress Windows UI 
+            // http://astrostatistics.psu.edu/datasets/R/html/utils/html/winMenus.html
+            return eval.ExecuteAsync(@"
+                rtvs:::suppress_function('bringToTop', 'grDevices');
```

Comment:
i meant this eval in the R code:
 replacement <- eval(parse(text = "function(...) { stop('Not supported') }"));